src/shared/components/FileDrop/FileDrop.style.ts

```diff
@@ -0,0 +1,161 @@
+import { breakpoints, colors, sizes, transitions, typography } from '@/shared/theme'
+import styled from '@emotion/styled'
+import Icon from '../Icon'
+import Text from '../Text'
+import { darken } from 'polished'
+import Button from '../Button'
+
+type DragAndDropAreaProps = {
+  isDragAccept?: boolean
+  isFileDialogActive?: boolean
+}
+
+type ProgressBarProps = {
+  progress?: number
+}
+
+export const FileDropWrapper = styled.div`
+  max-width: 640px;
+  height: 100%;
+`
+
+export const DragAndDropArea = styled.div<DragAndDropAreaProps>`
+  position: relative;
+
+  background-color: ${darken(0.16, colors.gray[600])};
+  width: 100%;
+  height: 0;
+  padding-top: 90%;
```

Comment:
There is not a lot of space to keep every element there. If I set padding-top back to 56,25 it will look something like this: 
![image](https://user-images.githubusercontent.com/51168865/110794532-239a5200-8276-11eb-864b-188a46c40f23.png)
Which I consider not ideal. I could lower even more margins, font-sizes and button padding, but it still looks kinda bad. 

I was thinking about three options here
1. I could remove button on mobile and allow mobile users to click on the droparea. I did it previously  
![image](https://user-images.githubusercontent.com/51168865/110795612-56911580-8277-11eb-84bb-c080d4c060b7.png)
2. Keep the present version with a squarish box
3. Remove something else on mobile view, maybe subtitle or icon?

What do you think? For me, the best option is the first one

Reply:
For 1 we'd need some logic based on the screen size though, right? Let's do the easiest thing for now (my guess is 2 since it's already done) and I'll bring this up with designers